docs/tutorials/quickstart.md

```diff
@@ -17,6 +17,12 @@ way to start it is to run it via a container:
     podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
 ```
 
+Alternatively, a script is available:
```

Comment:
Should: I think we should push this as the preferred way the developer runs rabbit rather than an alternative